<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40,37,58);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>Add Course</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>480</y>
      <width>841</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="backtodashboard_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backtodashboard_btn {
    background-color: none;
    border: 1.5px solid rgb(130,130,240) ;
	border-radius: 5px;
    color: white;
}

QPushButton#backtodashboard_btn:hover {
    background-color:  rgb(130,130,240);
	color: rgb(54, 54, 54)
    
}</string>
    </property>
    <property name="text">
     <string>Back to Dashboard</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Course Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coursecode_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255, 255, 255);border: 2px solid rgb(106, 98, 154); border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}

QLineEdit:hover{
color:rgb(255, 255, 255);border: 2px solid white; border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Course Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_course_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#add_course_btn {
    background-color: none;
    border: 1.5px solid rgb(36,199,103) ;
	border-radius: 10px;
    color: white;
}

QPushButton#add_course_btn:hover {
    background-color: rgb(36,199,103);
	color: rgb(54, 54, 54)
    
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_title_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255, 255, 255);border: 2px solid rgb(106, 98, 154); border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}

QLineEdit:hover{
color:rgb(255, 255, 255);border: 2px solid white; border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_course_info">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Product Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(236,77,82); text-align:center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Choose Course </string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_course_btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#delete_course_btn {
    background-color: none;
    border: 1.5px solid  rgb(234,77,86);
	border-radius: 10px;
    color: white;
}

QPushButton#delete_course_btn:hover {
    background-color: rgb(234,77,86);
	color: rgb(54, 54, 54)
    
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Choose Department</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dept_cbox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
border: 2px solid rgb(106, 98, 154);
border-radius: 8px;
background: rgb(34,31,44);
color: white;
padding: 12px;
font-size:15px;
}

/*
QComboBox:editable {
	//drop down menur border radius dewa hoile konay konay following color deha jaye
    background: red; 
	border-radius: 8px;
	color: white;
}*/


QComboBox:!editable, QComboBox::drop-down:editable {
background:  rgb(34,31,44); color: white;
}
QComboBox:!editable:hover, QComboBox::drop-down:!editable {
background:   rgb(34,31,44); color: aqua;
}
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	/* on state howar por main combobox er bgcolor ekhane change korte parbo*/
    background:  rgb(34,31,44);
	font-weight: 30px;
}
QComboBox:on { /* shift the text when the popup opens */
color:white;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px; /* arrow er separator ta left theke 30px duure */

    border-left-width: 2px;
    border-left-color: rgb(106, 98, 154);
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 8px; /* same radius as the QComboBox */
    border-bottom-right-radius: 8px;
	/* extra */
	/*border-bottom-left-radius: 8px;
	border-top-left-radius: 8px;*/

}
QComboBox::drop-down:hover {
   background: transparent;
}

QComboBox::down-arrow {
    /*image: url(/usr/share/icons/crystalsvg/16x16/actions/downdown.png);*/
	image: url(./downdown.png)
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
border: 2px solid rgb(106, 98, 154);
border-radius: 8px;
padding-left: 5px;
padding-right: 5px;
padding-top:5px;
padding-bottom:5px;
selection-background-color:  rgb(106, 98, 154);
color: white;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="course_cbox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
border: 2px solid rgb(106, 98, 154);
border-radius: 8px;
background: rgb(34,31,44);
color: white;
padding: 12px;
font-size:15px;
}

/*
QComboBox:editable {
	//drop down menur border radius dewa hoile konay konay following color deha jaye
    background: red; 
	border-radius: 8px;
	color: white;
}*/


QComboBox:!editable, QComboBox::drop-down:editable {
background:  rgb(34,31,44); color: white;
}
QComboBox:!editable:hover, QComboBox::drop-down:!editable {
background:   rgb(34,31,44); color: aqua;
}
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	/* on state howar por main combobox er bgcolor ekhane change korte parbo*/
    background:  rgb(34,31,44);
	font-weight: 30px;
}
QComboBox:on { /* shift the text when the popup opens */
color:white;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px; /* arrow er separator ta left theke 30px duure */

    border-left-width: 2px;
    border-left-color: rgb(106, 98, 154);
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 8px; /* same radius as the QComboBox */
    border-bottom-right-radius: 8px;
	/* extra */
	/*border-bottom-left-radius: 8px;
	border-top-left-radius: 8px;*/

}
QComboBox::drop-down:hover {
   background: transparent;
}

QComboBox::down-arrow {
    /*image: url(/usr/share/icons/crystalsvg/16x16/actions/downdown.png);*/
	image: url(./downdown.png)
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
border: 2px solid rgb(106, 98, 154);
border-radius: 8px;
padding-left: 5px;
padding-right: 5px;
padding-top:5px;
padding-bottom:5px;
selection-background-color:  rgb(106, 98, 154);
color: white;
}
</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="dept_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255, 255, 255);border: 2px solid rgb(106, 98, 154); border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}

QLineEdit:hover{
color:rgb(255, 255, 255);border: 2px solid white; border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Enter Department Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>Update/Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="update_text_courseno">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255, 255, 255);border: 2px solid rgb(106, 98, 154); border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}

QLineEdit:hover{
color:rgb(255, 255, 255);border: 2px solid white; border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="update_text_dept">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255, 255, 255);border: 2px solid rgb(106, 98, 154); border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}

QLineEdit:hover{
color:rgb(255, 255, 255);border: 2px solid white; border-radius:10px;padding-left:15px;padding-right:15px;background : rgb(34,31,44);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#update_btn_2{
    background-color: none;
    border: 1.5px solid  rgb(234,77,86);
	border-radius: 10px;
    color: white;
}

QPushButton#update_btn_2:hover {
    background-color: rgb(234,77,86);
	color: rgb(54, 20, 54)
    
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="update_btn_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#update_btn_3{
    background-color: none;
    border: 1.5px solid  rgb(234,77,86);
	border-radius: 10px;
    color: white;
}

QPushButton#update_btn_3:hover {
    background-color: rgb(234,77,86);
	color: rgb(54, 20, 54)
    
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Feedback</string>
   </property>
  </action>
  <action name="actionLog_out">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionLog_out_2">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
